packages/flutter_tools/test/host_cross_arch.shard/ios_content_validation_test.dart

```diff
@@ -172,6 +173,31 @@ void main() {
             expect(vmSnapshot.existsSync(), buildMode == BuildMode.debug);
           });
 
+          testWithoutContext('App.framework Info.plist contains correct MinimumOSVersion', () {
+            final File templateInfoPlist = fileSystem.file(
+              fileSystem.path.join(projectRoot, 'ios', 'Flutter', 'AppFrameworkInfo.plist'),
+            );
+            expect(templateInfoPlist, exists);
+            final String templateContents = templateInfoPlist.readAsStringSync();
+            expect(templateContents, isNot(contains('MinimumOSVersion')));
+
+            final File appFrameworkInfoPlist = outputAppFramework.childFile('Info.plist');
+            expect(appFrameworkInfoPlist, exists);
+
+            final expectedMinimumOSVersion = FlutterDarwinPlatform.ios
+                .deploymentTarget()
+                .toString();
+
+            final ProcessResult result = processManager.runSync(<String>[
+              'plutil',
+              '-p',
+              appFrameworkInfoPlist.path,
+            ]);
+
+            expect(result.exitCode, 0);
+            expect(result.stdout, contains('"MinimumOSVersion" => "$expectedMinimumOSVersion"'));
```

Comment:
Why do you need to use `plutil`, why not just check just the contents using `readAsStringSync`?